docs/reference/enums.mdx

```diff
@@ -5,6 +5,19 @@ category: Reference
 
 # Enums
 
+## `MessageState`
+
+The state of a message.
```

Comment:
Are there a defined set of possible states a message can be in or is it open-ended? If it's the latter, might be a good idea to list them there.